src/vm_cpu.rs

```diff
@@ -969,6 +970,29 @@ impl<'vcpu, 'pages, 'host, T: GuestStagePagingMode> ActiveVmCpu<'vcpu, 'pages, '
                         Ok(0)
                     }
                 }
+                CSR_HGATP => {
+                    let vmid_len = PerCpu::this_cpu().vmid_tracker().get_vmid_len();
+                    let mode = self.active_pages().get_hgatp_mode();
```

Comment:
The generic `T` for guest stage paging mode is in scope here, so no need for a method call.

Reply:
Fixed.